config/clusters/2i2c-aws-us/neurohackademy.values.yaml

```diff
@@ -107,6 +107,12 @@ jupyterhub:
           extra_resource_limits:
             nvidia.com/gpu: "1"
           init_containers: *init_containers
+      - display_name: R machine
+        slug: R
+        description: "Start a container with R available"
+        image:
```

Comment:
Does this need to be nested in a `profile_options` block? That's my guess just based on reading the profile for the GPU instance.

Reply:
Ah that is the correct diagnosis. I suppose this is the negative of looking at this while on my phone. Sorry for not catching it

Reply:
Thanks! See #4513